### PR TITLE
listed former contributors in emeritus section

### LINKED ISSUE #108: LDQ shoutout
Loki should be acknowledged in some form for his past contributions to Ilios. Somewhere on the [team page](https://www.iliosproject.org/team/) would be the obvious choice, perhaps in an "emeritus" section or the likes with other former contributors.

### PR DESCRIPTION
**Pull request by stopfstedt**
selection_446

fixes #108

### CODE REVIEW

File: team.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ topLevel: true
 - Geoffrey Boushey (@boushey), Applications Programmer
 - Dave Leonard (@dartajax), User Support
 
+### Emeritus
```
Comment (saschaben) [inline team.md]: We probably don't want to actually create a segment like this, rather just keep the separate blocks of names as they are now, and add a new block at the bottom made of Loki & Rolf.

File: team.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ topLevel: true
...
+
+- Loki der Quaeler (@quaeler), Lead Applications Programmer
```
Comment (saschaben) [inline team.md]: better to list as `Loki der Quaeler (@quaeler), Lead Developer (emeritus)

File: team.md [hunk 3]
```diff
@@ -13,6 +13,10 @@ topLevel: true
...
+- Rolf Mortensen (@rolfsf), Contributor (UX/UI)
```
Comment (saschaben) [inline team.md]: better to list as `Rolf Mortensen (@rolfsf), Usability Designer` (emeritus)

**Comment by saschaben** [comment]
please see inline comments. something more like

team 2018-01-02 14-01-18

File: team.md (hunk 1 above)
Comment (Trott) [inline team.md]: I think you'd want to say _Emeriti_ here as that's the plural. That will make it consistent with _Contributors_ above and _Sponsors_ below.

**Pull request merged**